lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_workspace_runtime.rb

```diff
@@ -63,7 +67,7 @@ def self.instantiate_with_user(uri, user, attrs)
 
       workspace
     rescue MiqAeException => err
-      $miq_ae_logger.error(err.message, :resource_id => miq_request_id)
+      workspace.logger.error(err.message, :resource_id => miq_request_id)
```

Comment:
did the `workspace.logger` allow us to no longer pass `:resource_id`?

Reply:
Correct, https://github.com/ManageIQ/manageiq-automation_engine/pull/570/files#diff-33fc5f7f8206b817c0ca3d4ad954ddb6c20cffd0aafc8c6b5c1966fb40804ecbR32
So we pass the resource_id during the initializer not on every log invocation

Reply:
so we can change this line?
```suggestion
      workspace.logger.error(err.message)
```